app/ts/utils/ethereumUtils.ts

```diff
@@ -76,6 +76,17 @@ export function bigintToDecimalStringWithUnknown(value: bigint | undefined, powe
 	return value === undefined ? '?' : bigintToDecimalString(value, power, maxDecimals)
 }
 
+export const isPracticallyInfinite = (value: bigint, power: bigint) => {
+	const googol = 10n ^ 100n
+	return value / (10n ** power) > googol
```

Comment:
Along with my comment below, we also need to get rid of the `/10n**power` because that makes the value on the lefthand side even smaller.
```suggestion
	const veryBig = 10n ^ 60n
	return value > veryBig
```
```suggestion
	const veryBig = 2n^250n
	return value > veryBig
```